src/runtime_src/driver/zynq/drm/zocl/sched_exec.c

```diff
@@ -545,9 +548,18 @@ configure(struct sched_cmd *cmd)
 	}
 
 	for (i  = 0; i < exec->num_cus; i++) {
+		if (is_valid_apts(zdev, cfg->data[i]) == -EINVAL) {
```

Comment:
The aperture list contain CUs and Debug IPs for now. This is not the best solution. Let's refine this once we have a general solution to support variety apertures.